app/services/service_offering/add_to_portfolio_item.rb

```diff
@@ -1,6 +1,7 @@
 module ServiceOffering
   class AddToPortfolioItem
     IGNORE_FIELDS = %w[id created_at updated_at portfolio_id tenant_id].freeze
+    XML_HEADER = "<?xml version=\"1.0\"?>".freeze
```

Comment:
Should this constant be a part of the `Catalog::CreateIcon` service?

Reply:
@lindgrenj6 If this is ImageMagick specific we should have a way to convert the our SVG data to ImageMagick data in a separate class and when we decide to change to using some other change we just change that class. This way the class can hold things relevant to ImageMagick.

Reply:
Well it's more along the lines of the fact that ImageMagick uses libxml2 to decode SVG documents - the SVG that topology sends back is just the `<svg></svg>` snippet rather than a well formed XML Document. 

We're decoding it on the backend to figure out what filetype it is, and ImageMagick blows up without the XML Header tag. So like all we can do is rescue on exception, then add the header and try again. OR just know that topology doesn't send us the header and just always add it on the svg docs that come back from topology.